### PR TITLE
Add checksum for lwtnn@2.13

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/lwtnn/package.py [hunk 1]
```diff
@@ -14,9 +14,10 @@ class Lwtnn(CMakePackage):
 
     maintainers = ['haralmha']
 
+    version('2.13', sha256='d5c1cede00f9d9cc0a25f070f744999453aec477dae6ef02701c11543c5b9553')
     version('2.12.1', sha256='b820e698d4ed60737e646ca87a42354e8ac548403348b7f2940e8fda1c0f8203')
     version('2.10', sha256='bf84b290c44da582226344b0d5febf7fdbd1cbdee94fcc8bcac972c7355564ed')
 
     depends_on('boost@1.54:')
-    depends_on('eigen@:3.3.99', when='@2.11:')
+    depends_on('eigen@:3.3.99', when='@2.11:2.12')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/lwtnn/package.py]: Can you add a comment to explain why `eigen` has an upper bound? Also (minor), can you invert the `eigen` dependencies so they read:
```python
depends_on('eigen')
depends_on('eigen@:3.3.99', when='@2.11:2.12')
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a couple of minor requests, otherwise LGTM

File: var/spack/repos/builtin/packages/lwtnn/package.py (hunk 1 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/lwtnn/package.py]: @alalazo done.

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
Giving @haralmha  some time to comment, but feel free to ping me in a couple of days for a merge.

**Review by haralmha — APPROVED** [review]

**Pull request merged**